modules/jolt_physics/objects/jolt_object_3d.cpp

```diff
@@ -137,6 +138,12 @@ bool JoltObject3D::can_interact_with(const JoltObject3D &p_other) const {
 }
 
 String JoltObject3D::to_string() const {
+	constexpr char fallback_name[] = "<unknown>";
```

Comment:
I'm not opposed to it per se, but it's a syntax I haven't seen before in the codebase. What's the advantage of doing this vs `const String fallback_name = "<unknown>";`?

Reply:
I mostly wanted to avoid the heap allocation for the "happy path", and somehow got it into my head that this was closer to a duplicated literal than `const char* fallback_name = "<unknown>"`, which does not seem to be the case.

All very silly in this context either way.